Assets/_Laz/Scripts/UserInterface/DebugBehaviour/DebugLazMovement/DebugMovementParametersEditor.cs

```diff
@@ -0,0 +1,312 @@
+using UnityEngine;
+
+namespace Laz
+{
+    [RequireComponent(typeof(RectTransform))]
+    public class DebugMovementParametersEditor : MonoBehaviour
+    {
+        private const float HEADER_HEIGHT_MULTIPLIER = 0.1f;
+        private const float BODY_HEIGHT_MULTIPLIER = 0.7f;
+        private const float FOOTER_HEIGHT_MULTIPLIER = 0.15f;
+        private const float HEADER_PADDING_MULTIPLIER = 0.03f;
+
+        private const float MIN_CURVATURE_RATE = 0.005f;
+        private const float MAX_CURVATURE_RATE = 0.1f;
```

Comment:
if LazMovementPropertyScriptableObject  changes the curvature rate, this will need changing too i'm guessing? 

wondering if there's a better way to reference it from the scriptable object